scripts/hash_password

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import argparse
+import bcrypt
+import getpass
+
+bcrypt_rounds=12
```

Comment:
What happens if we use a different number of rounds? The number of rounds can be configured in the config iirc.